### ISSUE TITLE
[BITV] 9.4.1.2/7.4 - The "TypeError" accordion buttons in the table are not accessible to users of assistive technology, as both the appropriate role of "Button" and the aria-expanded attribute are missing.

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37132]
For more information, see https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/. (1)

- [ ] Fix
- [ ] Backport

![414c728086658dfdb595a873bd04deeb](https://user-images.githubusercontent.com/1315170/223745965-fac01f81-489d-4ca8-85b4-aacd4d169131.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v7-n4

</details>
